.travis.yml

```diff
@@ -13,12 +13,18 @@ matrix:
       env: TESTS=blocks FLOATX=float32
     - python: 2.7
       env: TESTS=blocks FLOATX=float32 DB=sqlite
+    - python: 2.7
+      env: TESTS=blocks FLOATX=float32 DB=mimir
     - python: 3.4
       env: TESTS=blocks FLOATX=float64
     - python: 2.7
+      env: TESTS=blocks FLOATX=float32 DB=mimir
```

Comment:
I agree, that's too much. I'll remove some tests after I'll be sure that everything works.